src/components/Card/index.tsx

```diff
@@ -44,7 +44,7 @@ const Card: FC<Props> = ({
 
   return (
     <div
-      className={classNames.card}
+      className={`classNames.card ${title.link ? "classNames.pointer" : ""}`} 
```

Comment:
Great, you got the gist 🥳 
But your solution won't apply class `card` because it'll be converted in the string `classNames.card`.

You might want to change it to something like below,
```
className={`${classNames.card} ${title.link && classNames.pointer}`}
```

Reply:
Got it! Thank you

Reply:
Cool 😄 
Just one suggestion, could we use the `.links` (line 78 of scss) instead of creating a totally separate and new class?